draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -3034,7 +3034,7 @@ directly from the indifferentiability of H.
 For case (3), i.e., for H a Merkle-Damgaard hash function, indifferentiability
 follows from {{CDMP05}}, Theorem 3.5.
 In particular, expand\_message\_xmd computes b\_0 by prefixing the message
-with one block of 0-bytes plus auxiliary information (length, counter, and DST).
+with one block of zero bytes plus auxiliary information (length, counter, and DST).
```

Comment:
```suggestion
- with one block of zero bytes plus auxiliary information (length, counter, and DST).
+ with one block of zeros plus auxiliary information (length, counter, and DST).
```

Not sure, if the sentence can be interpreted as: "a block that has a length of zero bytes", instead of a block with all zeros.

Reply:
Agreed that it would be good to avoid the garden path, "a block of bytes with length zero".